bin/.travis/test.sh

```diff
@@ -33,6 +33,11 @@ if [ "$EZ_VERSION" = "" ]; then
     EZ_VERSION="^2.4"
 fi
 
+if [ "$EZ_VERSION" = "^2.4" ]; then
+    PHP_VARIANT='latest-node'
+else
+    PHP_VARIANT='latest'
+fi
```

Comment:
why do this? it will break as soon as we set ^2.5 or something else, why not just use latests-node further down?

Reply:
@vidarl this remark still stands

Reply:
Fixed